code/modules/mob/living/simple_animal/hostile/megafauna/chaos_marine.dm

```diff
@@ -0,0 +1,510 @@
+/mob/living/simple_animal/hostile/megafauna/chaos_marine
+	name = "chaos marine"
+	desc = "A servant of dark gods, chaos marine is always in state of eternal battle."
+	health = 2000
+	maxHealth = 2000
+	icon_state = "chaos_marine"
+	icon_living = "chaos_marine"
+	icon = 'icons/Fulpicons/megafauna.dmi'
+	faction = list("chaos")
+	mob_biotypes = MOB_ORGANIC|MOB_HUMANOID
+	light_color = "#FFFFFF"
+	movement_type = GROUND
+	speak_emote = list("roars")
+	speed = 6
+	light_range = 6
+	move_to_delay = 4
+	projectiletype = /obj/projectile/bullet/chaos_bomb
+	projectilesound = 'sound/weapons/gun/l6/shot.ogg'
+	ranged = TRUE
+	ranged_cooldown_time = 20
+	vision_range = 10
+	damage_coeff = list(BRUTE = 1, BURN = 0.5, TOX = 0.5, CLONE = 0.5, STAMINA = 0, OXY = 0.5)
+	loot = list(/obj/item/nullrod/scythe/talking/chainsword/chaos, /obj/item/clothing/suit/space/hardsuit/ert/paranormal/berserker/chaos)
+	crusher_loot = list(/obj/item/nullrod/scythe/talking/chainsword/chaos, /obj/item/clothing/suit/space/hardsuit/ert/paranormal/berserker/chaos)
```

Comment:
c'mon, throw in the blood drunk miner crusher trophy or something

Reply:
I wanted to add something unique